File: API/src/main/java/entities/Professor.java
package entities;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.*;

import java.util.List;

@Table(name="professor")
@Entity(name = "professor")
public class Professor {
    @Id
    private String email;
    private String username;
    private String password;
    private String school;
    @ManyToMany
    @JoinTable(name = "Professor_Class", joinColumns = @JoinColumn(name = "Class_id"), inverseJoinColumns = @JoinColumn(name = "Professor_id"))
    @JsonIgnoreProperties("professores")
    private List<Class> classes;
}

File: eduwise-front/react_api/Models/Professor.cs
using System.ComponentModel.DataAnnotations;

namespace react_api.Models
{
    public class Professor
    {
        public int id {get;set;}
        public string? email {get; set;}
        public string? username {get;set;}
        public string? password {get;set;}
        public string? school {get;set;}
    }
}